lib/modules/quarantineHide.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+
+import { Module } from '../core/module';
+import { $ } from '../vendor';
+
+export const module: Module<*> = new Module('quarantineHide');
+
+module.moduleName = 'qhideName'; // i18n
+module.category = 'appearanceCategory';
+module.description = 'qhideDesc'; // i18n
+module.options = {
+	hideFlair: {
+		title: 'qhideFlairTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideFlairDesc', // i18n
+	},
+	hideSidebar: {
+		title: 'qhideSidebarTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideSidebarDesc', // i18n
+	},
+	fixHeaderImg: {
+		title: 'qhideFixHeaderImgTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideFixHeaderImgDesc', // i18n
+	},
+	hideWarning: {
+		title: 'qhideWarningTitle', // i18n
+		type: 'boolean',
+		value: false,
+		description: 'qhideWarningDesc', // i18n
+	},
+};
+
+module.include = ['linklist'];
+
+module.go = () => {
+	if (module.options.hideFlair.value) {
+		removeFlair();
+		window.addEventListener('neverEndingLoad', removeFlair);
+	}
+	if (module.options.hideSidebar.value) {
+		const $notice = $('.quarantine-notice');
```

Comment:
To determine if this is a quarantined subreddit, it would probably be cleaner and more robust to check if `document.body.classList.has('quarantine')`